projects/smart-table/src/lib/components/column-selector/column-selector.component.html

```diff
@@ -0,0 +1,28 @@
+<div auiFlyout size="small" align="right">
+  <button class="a-button-outline has-icon"
+          [attr.aria-label]="(configuration$ | async)?.options.translations?.openColumnSelector || 'Toon/verberg kolommen'"
+          [title]="(configuration$ | async)?.options.translations?.openColumnSelector || 'Toon/verberg kolommen'"
+          auiFlyoutAction>
+    <span class="fa fa-columns"></span>
+  </button>
+
+  <div auiFlyoutZone>
+    <div class="u-margin-xs">
+      <ul class="a-list a-list--reset aui-smart-table__column-checklist u-margin-bottom-xs u-text-left"
+          cdkDropList (cdkDropListDropped)="updateColumnsSortIndex($event)" aria-dropeffect="move">
+        <li aria-grabbed="true" cdkDrag class="a-list__item"
+            *ngFor="let column of (pendingColumnOperation$ | async); index as i">
+          <div class="a-input__checkbox">
+            <input type="checkbox" [disabled]="!(column.canHide !== false)" id="{{ 'checkbox--' + i + '--' + column.key }}"
```

Comment:
Waarom niet gewoon [disabled]="!column.canHide"? Of kunnen er ook andere waardes in gezet worden dan true of false?

Reply:
Omdat canHide ook undefined of andere waarden kan hebben (of ook gwn niet bestaande kan zijn). Hij moet echt expliciet op false gezet worden om als false gezien te zijn.